index.js

```diff
@@ -60,14 +63,15 @@ async function fastifyRoutesStats (fastify, opts) {
       ? reply.context.config.statsId
       : request.raw.url
 
-    const id = request.raw.id
```

Comment:
```js
  fastify.addHook('onSend', function (request, reply, _, next) {
    if (request[decoratorName]) {
      const routeId = reply.context.config.statsId
        ? reply.context.config.statsId
        : request.raw.url

      const id = request.id
      const key = `${ROUTES}${request.raw.method}|${routeId}`

      performance.mark(ONSEND + id)
      performance.measure(key, ONREQUEST + id, ONSEND + id)
      performance.clearMarks(ONSEND + id)
      performance.clearMarks(ONREQUEST + id)
    } else {
      fastify.log.error('missing request mark')
    }

    next()
  })
```

Reply:
we dont need all the variables if the request is missing the decorated attribute.

Reply:
Actually... why do I even used a ternary operation?!
the folowing should be equivalent

`const routeId = reply.context.config.statsId || request.raw.url`